CSS/style.css

```diff
@@ -0,0 +1,140 @@
+body {
+	max-width: 1440px;
+	overflow: visible;
+	margin-left: auto;
+	margin-right: auto;
+}
+.header {
+	position: absolute;
+	display: flex;
+	justify-content: space-between;
+	overflow: visible;
+	margin: 30px 186px 100px 186px;
+}
+.logo {
+	margin: 9.5px 149px 9.5px 0px;
+}
+.link__group-1 {
+	display: inline-flex;
+}
+
+.nav__link {
+	display: inline-block;
+	margin: 14px 24.5px 26px 24.5px;
+	font-family: DM Sans;
+	font-style: normal;
+	font-weight: bold;
+	font-size: 15px;
+	line-height: 20px;
+	letter-spacing: -0.468947px;
+	color: #696871;
+	text-decoration: none;
+}
+.start__form {
+	margin-left: 133.5px;
+}
+.start__button {
+	font-family: DM sans, sans-serif;
+	height: 60px;
+	width: 211px;
+	border-radius: 10px;
+	background-color: #ff7143;
+	border: none;
+	cursor: pointer;
+
+	font-family: DM Sans;
+	font-style: normal;
+	font-size: 17px;
+	line-height: 22px;
+	text-align: center;
+	letter-spacing: 0.531474px;
+	color: #ffffff;
+}
+.start__button:active {
+	background-color: #ff4144;
+}
+.nav__link {
+	position: relative;
+}
+.nav__link:hover {
+	color: #19191b;
+}
+.nav__link:hover::before {
+	transform: scaleX(1);
+	transition: 0.3s;
+}
+.nav__link::before {
+	content: "";
+	width: 100%;
+	height: 2px;
+	background-color: #5454d4;
+	position: absolute;
+	left: 0px;
+	bottom: -10px;
+	transform: scaleX(0);
+}
+
+.hero {
+	display: inline-flex;
+	width: 1146px;
+	height: 644px;
+	margin: 190px 100px 0 184px;
+}
+
+.hero__text1 {
+	position: absolute;
+	height: 231px;
+	width: 462px;
+	margin-top: 92px;
+
+	font-family: DM Sans;
+	font-style: normal;
+	font-weight: bold;
+	font-size: 80px;
+	line-height: 77px;
+	letter-spacing: -2.50105px;
+	color: #19191b;
+}
+.hero__text2 {
+	position: absolute;
+	height: 90px;
+	width: 459px;
+	margin-top: 353px;
+
+	font-family: Mulish, sans-serif;
+	font-style: normal;
+	font-weight: normal;
+	font-size: 20px;
+	line-height: 30px;
+	letter-spacing: -0.625263px;
+	color: #696871;
+}
+.hero__image {
+	margin: 0px 30px 0px 536px;
+}
+.hero__link1 {
+	position: absolute;
+	margin-top: 493px;
+
+	font-family: DM sans, sans-serif;
```

Comment:
your need tiks

Reply:
What it means?

Reply:
https://developer.mozilla.org/ru/docs/Web/CSS/font-family